site/data/docs/main-toc.yml

```diff
@@ -66,21 +66,35 @@ toc:
   - title: Guides
     subfolderitems:
       - page: AWS with NLB
-        link: /guides/deploy-aws-nlb/
+        url: /guides/deploy-aws-nlb/
```

Comment:
TODO finalize titles

Reply:
Updated these to mostly match the individual page titles, they're now a bit longer and they wrap, but I think it's OK overall.